crypto/fipsmodule/sha/sha3.c

```diff
@@ -110,12 +110,16 @@ uint8_t *SHAKE256(const uint8_t *data, const size_t in_len, uint8_t *out, size_t
 }
 
 // FIPS202 APIs manage internal input/output buffer on top of Keccak1600 API layer
+// FIPS202_Reset zero's |ctx| fields.
```

Comment:
NP: It seems that the "FIPS202 APIs manage ..." comment might be better placed inside a multi-line `/* ... */` comment as it relates to several functions below.

Reply:
Thanks! Just updated.